shared/actions/config/index.tsx

```diff
@@ -354,14 +358,6 @@ function* maybeDoneWithLogoutHandshake(state: Container.TypedState) {
   }
 }
 
-let lastTab: Tabs.Tab | undefined
```

Comment:
this is entirely handled using `useState` when we go from loggedIn => out => in now